src/reducers/scene.js

```diff
@@ -4,9 +4,8 @@ export const DEF_SETTINGS = {
     skyColor: "white",
     floorColor: "#222",
     camConfig: 0,
-    showCoordHelper: true,
```

Comment:
`showCoordHelper` and `showFloor` should not have their default values be changed.